app/src/components/market/market_home_container.tsx

```diff
@@ -90,7 +92,7 @@ const MarketHomeContainer: React.FC = () => {
       variables: {
         skip: fetchedMarkets.fixedProductMarketMakers.length,
       },
-      updateQuery: (prev, { fetchMoreResult }) => {
+      updateQuery: (prev: any, { fetchMoreResult }) => {
```

Comment:
I don't know why this gives me an error of type unknown instead of using any implicitly.